### PR TITLE
Sprint 20

### PR DESCRIPTION
**Pull request by ChristianMurphy**

### CODE REVIEW

File: web-app/js/plugins/jquery.custom.draggable.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+// src: https://css-tricks.com/snippets/jquery/draggable-without-jquery-ui/
```
Comment (ChristianMurphy) [inline web-app/js/plugins/jquery.custom.draggable.js]: @rahulparekh why did we customize `draggable` rather than using `jquery-ui` `draggable`?

**Comment by ChristianMurphy** [comment]
@nishantk12 why do all the line endings need to be changed?

**Comment by nishantk12** [comment]
Can you be more elaborative when you say line endings? Will check

**Comment by ChristianMurphy** [comment]
> Can you be more elaborative when you say line endings? Will check
> @nishantk12 

The [new line character](https://en.wikipedia.org/wiki/Newline) was set to `lf` but changed to `cr+lf`.
and the diff shows 11,000 lines of code changed by this.

**Comment by nishantk12** [comment]
Ok, so first of all I'll ask you to believe me that I didn't go to every 
line and change the new line chara from lf(line feed) to crlf(carriage 
return line feed). There was some file handling standard which did this 
change.

I googled for a while and found that this can be due to difference in OS 
which are being used. But then this was not happening earlier. So, I 
suspect that this is the first time I used GGTS to commit changes, 
staged files and then doing the merge. May be this has caused the 
changes.

------ Original Message ------

**Comment by ChristianMurphy** [comment]
> Ok, so first of all I'll ask you to believe me that I didn't go to every
line and change the new line
> @nishantk12 

I believe you, some git setups have `auto-crlf` set, while others do not.

> I googled for a while and found that this can be due to difference in OS
which are being used.

OS's can also do that

I'll look into setting a `gitattributes` file so that this will be normalized across OS's

**Comment by nishantk12** [comment]
Thanks Christian. This is an interesting point you bought up. Adds to 
leaning of us, mine for sure.

you can refer this link for some interesting feeds. 
http://www.hanselman.com/blog/YoureJustAnotherCarriageReturnLineFeedInTheWall.aspx

------ Original Message ------

**Comment by ChristianMurphy** [comment]
No Problem! always glad to help :smile: 

Set the `.gitattribute` and `.editorconfig` to always use `lf`.

**Pull request merged**